stable/minecraft/Chart.yaml

```diff
@@ -1,5 +1,7 @@
 name: minecraft
-version: 0.2.0
+version: 0.2.1
+appVersion: 1.7.9
```

Comment:
Not sure how you found 1.7.9, but when I check the image (and it's a pretty rabbit hole), I get to 1.12.2 as being the latest. I recommend to use that, as it is at least correct for the time being.

Reply:
Thanks @timstoop 
So we use 1.12.2 for that. It's modified.